test/Spec.hs

```diff
@@ -3,13 +3,16 @@ module Main (main) where
 import System.IO (hSetEncoding, utf8)
 import Test.Hspec (hspec)
 
+import Test.HLint (hlintSpec)
+
 import qualified Test.Config as Config
 import qualified Test.Suggestion as Suggestion
 
 main :: IO ()
 main = do
   hspec Config.spec
   hspec Suggestion.spec
+  hspec hlintSpec
   hSetEncoding stdout utf8
   hSetEncoding stderr utf8
```

Comment:
I propose to refactor these tests a little bit:

1. Use 4 spaces
2. Move `hSetEncoding` at the beginning of `main`
3. Can put all unit tests inside single `do`-block, like

```haskell
hspec $ do
    Config.spec
    Suggestion.spec
    hlintSpec
```